djangocms_admin_style/sass/components/_popup.scss

```diff
@@ -74,13 +74,15 @@
     }
     #content,
     &.filebrowser #content {
-        margin: 20px 2% 0 !important;
+        margin: 20px 20px 0 !important;
     }
     #content .submit-row {
         margin: 20px -2% 0 !important;
```

Comment:
i think we hide submit-row, but shouldn't it still be `20px -20px 0 !important`?